### PR TITLE
deskpins: Add version 1.32

### LINKED ISSUE #4628: [Request ]Add DeskPins
Hello,

Could you please consider adding DeskPins?
https://efotinis.neocities.org/deskpins

Source code:
https://efotinis.neocities.org/downloads/efotinis-deskpins-e3caef22897e.zip

Latest release for Win 10:
https://efotinis.neocities.org/downloads/DeskPins-1.32-setup.exe

Requirements: Visual C++ redistributables:
https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

Thank you!

### PR DESCRIPTION
**Pull request by issaclin32**
- Closes #4628

**DeskPins** ([homepage](https://efotinis.neocities.org/deskpins/)) is a tool used to make any application topmost, that is, to keep it above all other windows.

Notes:

* **license**: ~The MIT license can be found at `license.txt` of [the source code package](https://efotinis.neocities.org/downloads/efotinis-deskpins-e3caef22897e.zip)~. Use "Freeware" instead.

* **persist** is not needed because DeskPins stores its config data at `HKCU:\SOFTWARE\Elias Fotinis\DeskPins`.

### CODE REVIEW

File: bucket/deskpins.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.32",
+    "description": "Make any application always on top.",
+    "homepage": "https://efotinis.neocities.org/deskpins",
+    "license": "MIT",
```
Comment (Ash258) [inline bucket/deskpins.json:5]: image

File: bucket/deskpins.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "url": "https://efotinis.neocities.org/downloads/DeskPins-1.32-setup.exe#/dl.7z",
+    "hash": "70bfd44e774837e52bc83f2c128de7d164251e513f6ebea4a70e2073e28ecd2a",
+    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Force -Recurse",
+    "bin": "deskpins.exe",
```
Comment (Ash258) [inline bucket/deskpins.json:9]: What are available parameters?

**Review by Ash258 — CHANGES_REQUESTED** [review]
image

File: bucket/deskpins.json (hunk 1 above)
Comment (issaclin32) [reply bucket/deskpins.json:5]: > Notes:
> * license: The MIT license can be found at license.txt of the source code package.

The author puts MIT license in the source code archive. Should I change *license* to "Freeware", or just keep it as "MIT"?

File: bucket/deskpins.json (hunk 2 above)
Comment (issaclin32) [reply bucket/deskpins.json:9]: Do you mean that `bin` is not needed when there are no command-line parameters available?

File: bucket/deskpins.json (hunk 1 above)
Comment (Ash258) [reply bucket/deskpins.json:5]: How is the source code license related to the released binary, when there is specific license for the final binary?

File: bucket/deskpins.json (hunk 2 above)
Comment (Ash258) [reply bucket/deskpins.json:9]: No... I mean: What are the command line parameters avaialable??

File: bucket/deskpins.json (hunk 1 above)
Comment (issaclin32) [reply bucket/deskpins.json:5]: OK. I understood. I will change *license* to "Freeware".

File: bucket/deskpins.json (hunk 2 above)
Comment (issaclin32) [reply bucket/deskpins.json:9]: I don't think there are any command-line parameters available.
I looked through the product page(https://efotinis.neocities.org/deskpins/index.html) and the manual (`DeskPins.chm` in the binary release), but cannot find any informations about command-line parameters.

Comment (Ash258) [reply bucket/deskpins.json:9]: Then remove it

File: bucket/deskpins.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "url": "https://efotinis.neocities.org/downloads/DeskPins-1.32-setup.exe#/dl.7z",
+    "hash": "70bfd44e774837e52bc83f2c128de7d164251e513f6ebea4a70e2073e28ecd2a",
+    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Force -Recurse",
+    "shortcuts": [
+        [
+            "deskpins.exe",
+            "DeskPins"
+        ]
+    ],
+    "suggest": {
```
Comment (Ash258) [inline bucket/deskpins.json:15]: Dependency suggest has to be before URL

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — CHANGES_REQUESTED** [review]
Remove checkver and autoupdate

**Review by Ash258 — APPROVED** [review]

**Comment by 7wells** [comment]
Thank you so much! 👍

**Pull request merged**